src/Core/Vipr.Core/CodeModel/OdcmCapabilities.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vipr.Core.CodeModel
+{
+    public class OdcmCapabilities
+    {
+        public bool Insertable { get; set; }
```

Comment:
Changed this to use a collection of OdcmCapability. Now Writers can iterate over this collection to get a list of capability. it also make adding new capabilities support a bit easy.